src/acceptor_pool.erl

```diff
@@ -77,6 +77,12 @@
 -export([format_status/2]).
 -export([terminate/2]).
 
+%% cached connections api
+-export([save_conn/5,
+         take_conn/2,
+         saved_conns/1,
+         num_conns/1]).
```

Comment:
only num_conns seems to be used outside of this module?

Reply:
fixed here: fc36e27aca1b789b222d5af9af757eebbdb948ae